packages/evm/contracts/Controller.sol

```diff
@@ -35,25 +35,31 @@ contract Controller is IController, Ownable {
     // List of allowed validators
     mapping (address => bool) public override isValidatorAllowed;
 
+    // Minimum number of validations allowed
+    uint8 public minValidations;
+
     /**
      * @dev Creates a new Controller contract
      * @param owner Address that will own the contract
      * @param solvers List of allowed solvers
      * @param executors List of allowed executors
      * @param proposalSigners List of allowed proposal signers
      * @param validators List of allowed validators
+     * @param _minValidations Minimum number of validations allowed
      */
     constructor(
         address owner,
         address[] memory solvers,
         address[] memory executors,
         address[] memory proposalSigners,
-        address[] memory validators
+        address[] memory validators,
+        uint8 _minValidations
     ) Ownable(owner) {
         for (uint256 i = 0; i < solvers.length; i++) _setAllowedSolver(solvers[i], true);
         for (uint256 i = 0; i < executors.length; i++) _setAllowedExecutor(executors[i], true);
         for (uint256 i = 0; i < proposalSigners.length; i++) _setAllowedProposalSigner(proposalSigners[i], true);
         for (uint256 i = 0; i < validators.length; i++) _setAllowedValidator(validators[i], true);
+        minValidations = _minValidations;
```

Comment:
Let's use an internal setter here so we emit an event properly

Reply:
Done [Fix min validations comments](https://github.com/mimic-protocol/contracts/pull/36/commits/131f65e76064112cb20d3dc33a3d6dd62bf93450)